package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "vite-plugin-i18next-loader",
-  "version": "2.0.2",
+  "name": "vite-plugin-i18next-loader-cjs",
+  "version": "2.0.4",
```

Comment:
Don't bump the version, `auto` will do that instead.  Also change back the name.

Reply:
Hey sure that makes sense!